scripts/setup_iso.py

```diff
@@ -76,6 +78,11 @@ def do_setup_repo(osname, osver, src, dest, link):
         os.system(syscall)
 
         shutil.copyfile(src+'/isolinux/'+vmlinuz, dstpath+'/'+vmlinuz)
+    elif osname == 'Ubuntu':
+        p = subprocess.Popen(['ls', src], stdout=subprocess.PIPE)
+        filenames = p.stdout.read().split('\n')
+        ignoreList=[f for f in filenames if f == 'ubuntu' or f == '']
```

Comment:
I'm afraid using the filename to screen out the soft link is not a good idea, since a file/folder that named with `ubuntu` may not always be the soft link. The "ls -l" command will tell the information whether it is a link or not